.dockerignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything
+**
+
+# Allowed files
```

Comment:
Interesting - not seen this before. What's it useful for?

Reply:
In our use case, we use container only for executing tasks. There is no need to include the code in each of the containers really from my point of view.